src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1104,6 +1103,155 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         }
     }
 
+    @Tidy
+    def "System doesn't reroute flow to more preferable path while updating portNumber/vlanId"() {
```

Comment:
I'd rename it `System doesn't rebuild path for the flow...` since reroute could be miss-leading here

Reply:
renamed